main.go

```diff
@@ -125,6 +125,8 @@ func (c *Collector) Collect(ch chan<- prometheus.Metric) {
 
 		resources, err := ac.listFromResourceGroup(target.ResourceGroup, target.ResourceTypes)
 		if err != nil {
+			log.Printf("Failed to get resources for resource group %s and resource types %s: %v",
```

Comment:
Is it a debug or a fatal ? I guess you're right just checking

Reply:
I used same log statement as other errors logging in the project.